app/src/main/res/values/dimens.xml

```diff
@@ -588,4 +588,13 @@
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">32dp</dimen>
 
+  <!-- Topic Practice Header-->
+  <dimen name="custom_match_parent">-1px</dimen>
+  <dimen name="topic_practice_header_container_width">@dimen/custom_match_parent</dimen>
```

Comment:
Never use a negative dimens value. If you are not able to merge mobile and tablet xmls then it is okay to keep two separate files for them.